chart/Chart.yaml

```
@@ -48,8 +48,8 @@ annotations:
   artifacthub.io/screenshots: |
     - title: DAGs View
       url: https://airflow.apache.org/docs/apache-airflow/2.2.5/_images/dags.png
-    - title: Tree View
-      url: https://airflow.apache.org/docs/apache-airflow/2.2.5/_images/tree.png
+    - title: Grid View
+      url: https://airflow.apache.org/docs/apache-airflow/2.2.5/_images/grid.png
```

```suggestion
    - title: Tree View
      url: https://airflow.apache.org/docs/apache-airflow/2.2.5/_images/tree.png
```

Thanks for being proactive, but this will need to wait until the next helm chart release after 2.3.0 launches.
Said another way, this is a follow up step after releasing a core Airflow version, when we make the new version the chart default.

---

So remove this change for now and do it in a later PR?

---

Yep exactly, when we hit this step in the release process:
https://github.com/apache/airflow/blob/main/dev/README_RELEASE_AIRFLOW.md#update-default-airflow-version-in-the-helm-chart